src/xcode/ENA/ENA/Source/Scenes/QRScanner/QRScannerCoordinator.swift

```diff
@@ -84,13 +84,16 @@ class QRScannerCoordinator {
 	private var traceLocationCheckinCoordinator: TraceLocationCheckinCoordinator?
 	private var exposureSubmissionCoordinator: ExposureSubmissionCoordinator?
 	private var onBehalfCheckinCoordinator: OnBehalfCheckinSubmissionCoordinator?
-	
+
+	private var fileScannerCoordinator: FileScannerCoordinator?
+
 	private func qrScannerViewController(
 		markCertificateAsNew: Bool,
 		markCoronaTestAsNew: Bool,
 		didDismiss: @escaping () -> Void
 	) -> UIViewController {
-		return QRScannerViewController(
+		var qrScannerViewController: QRScannerViewController!
+		qrScannerViewController = QRScannerViewController(
```

Comment:
a little bit ugly, but necessary because we refer qrScannerViewController in the presentFileScanner closure, right?
If not, I think we cannot return qrScannerViewController immediately but need to declare here a var so perhaps this will work?: 
```suggestion
		var qrScannerViewController = QRScannerViewController(
```

Reply:
This won't work :-(

Reply:
okay, just i thought.